### PR TITLE
`src/common/*transformations/` API1.0 clean up

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

File: src/common/low_precision_transformations/tests/batch_to_space_transformation.cpp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 #include <transformations/utils/utils.hpp>
 #include <transformations/init_node_info.hpp>
-#include <ngraph/opsets/opset2.hpp>
+// #include <ov/opsets/opset2.hpp>
```
Comment (ilya-lavrenov) [inline src/common/low_precision_transformations/tests/batch_to_space_transformation.cpp:14]: remove?

File: src/common/transformations/tests/utils/primitives_priority_test.cpp [hunk 2]
```diff
@@ -12,12 +12,11 @@
...
 
 #include "common_test_utils/ov_test_utils.hpp"
-#include "common_test_utils/test_common.hpp"
-#include "ie_ngraph_utils.hpp"
+// #include "common_test_utils/test_common.hpp"
 #include "openvino/core/model.hpp"
 #include "openvino/opsets/opset1.hpp"
 #include "transformations/rt_info/primitives_priority_attribute.hpp"
-#include "transformations/utils/utils.hpp"
+// #include "transformations/utils/utils.hpp"
```
Comment (ilya-lavrenov) [inline src/common/transformations/tests/utils/primitives_priority_test.cpp:19]: remove commented headers?

File: src/common/low_precision_transformations/tests/batch_to_space_transformation.cpp (hunk 1 above)
Comment (vurusovs) [reply src/common/low_precision_transformations/tests/batch_to_space_transformation.cpp]: Yes, thank you a lot!

File: src/common/transformations/tests/utils/primitives_priority_test.cpp (hunk 2 above)
Comment (vurusovs) [reply src/common/transformations/tests/utils/primitives_priority_test.cpp]: Yes, thank you a lot!

**Review by ilya-lavrenov — APPROVED** [review]

**Review by itikhono — APPROVED** [review]

**Pull request merged**